WoWsShipBuilder.ViewModels/Other/ShipComparisonDataSections.cs

```diff
@@ -1,5 +1,68 @@
-﻿namespace WoWsShipBuilder.ViewModels.Other;
+﻿using WoWsShipBuilder.Core.Localization;
 
+namespace WoWsShipBuilder.ViewModels.Other;
+
+
+public static class ShipComparisonDataSection
```

Comment:
Extract this class into a separate file and rename it. ShipComparisonDataSectionExtensions or DataSectionExtensions would be appropriate